rules/Proj0017.md

```diff
@@ -0,0 +1,26 @@
+# Proj0017: Can't create alias for static using directive
+`<Using>` nodes can not be both marked as `Static` and
+be an `Alias`. For more info
+[see](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/using-directive-errors?f1url=%3FappId%3Droslyn%26k%3Dk(CS8085)#restrictions-on-using-aliases).
+
+## Non-compliant
+``` XML
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <ProjectReference Include="NamespaceA.Type" Static="True" Alias="AType" />
```

Comment:
Does MS Build allow this?

Reply:
Yes MSBuild allows this, but this results in generated code that does not compile. I'm not entirely sure if this rule should be a warning or an error (since it will inevitably result in the linked compile error).